README.md

```diff
@@ -1,2 +1,48 @@
 # Intention-Timer
-Turing Mod 1 Project 5, building an application that allows a user to set goals for their health and productivity, tied to an amount of time.
+
+## Contributors
+
+* Connor Anderson-Larson
+* Jim Charnesky
+* Joel Thomas
+
+## Technologies Used:
+
+* JavaScript
+* CSS
+* HTML
+
+## Deploy Link:
+
+* [Intention Timer](https://connorandersonlarson.github.io/Intention-Timer/)
+
+## How to Use:
```

Comment:
Consider adding installation instructions/ instructions on how to open the project. It may seem obvious how to now, but as you learn more technologies and your projects get more complex, it might not be as obvious to a visitor checking out your project!